storage/innobase/handler/ha_innodb.cc

```diff
@@ -7343,7 +7349,11 @@ wsrep_store_key_val_for_row(
 								cs->mbmaxlen),
 							&error);
 				}
-				memcpy(sorted, src_start, true_len);
+
+				if (src_start) {
+					memcpy(sorted, src_start, true_len);
+				}
+
```

Comment:
same applies to this, as the Marko's comment above, assert is better than circumventing the potential mismatch of buffer pointer and buffer length. 
Here, the src_start refers to function parameter record, i suppose it should not be nullptr in any case